README.rst

```diff
@@ -425,6 +428,135 @@ view (only when using HTTP POST).
+|  value type  |                   ``int``                                |
++--------------+----------------------------------------------------------+
+|  field_name  |                ``free_memory``                           |
++--------------+----------------------------------------------------------+
+|configuration |                  ``memory``                              |
++--------------+----------------------------------------------------------+
+|related_fields| ``total_memory``, ``buffered_memory``, ``shared_memory`` |
++--------------+----------------------------------------------------------+
+|    charts    |                  ``memory``                              |
++--------------+----------------------------------------------------------+
+
+.. figure:: https://github.com/openwisp/openwisp-monitoring/master/docs/memory.png
+  :align: center
+
+``CPU Load``
+~~~~~~~~~~~~
+
++--------------+-------------------------------------------------+
+|     name     |             ``Resources: Load``                 |
++--------------+-------------------------------------------------+
+|     key      |                   ``load``                      |
++--------------+-------------------------------------------------+
+|  value type  |                   ``float``                     |
++--------------+-------------------------------------------------+
+|  field_name  |                   ``load_1``                    |
++--------------+-------------------------------------------------+
+|configuration |                   ``load``                      |
++--------------+-------------------------------------------------+
+|related_fields|             ``load_5``, ``load_15``             |
++--------------+-------------------------------------------------+
+|    charts    |                  ``load``                       |
++--------------+-------------------------------------------------+
+
+``Disk Usage``
+~~~~~~~~~~~~~~
+
++--------------+-------------------------------------------------+
+|     name     |              ``Resources: Disk``                |
++--------------+-------------------------------------------------+
+|     key      |                   ``disk``                      |
++--------------+-------------------------------------------------+
+|  value type  |                   ``float``                     |
++--------------+-------------------------------------------------+
+|  field_name  |                 ``used_disk``                   |
++--------------+-------------------------------------------------+
+|configuration |                   ``disk``                      |
++--------------+-------------------------------------------------+
+|    charts    |                   ``disk``                      |
++--------------+-------------------------------------------------+
+
+.. figure:: https://github.com/openwisp/openwisp-monitoring/master/docs/disk_usage.png
```

Comment:
Does this work? https://drive.google.com/file/d/1BgxmXEkdEgCWlWASALZB2a3WeZad0wXw/view?usp=sharing

Reply:
The link needs to be edited slightly but it works, added the commit.